### DISCUSSION TITLE
is timezone in points/{latitude},{longitude} that of location or office?

### DISCUSSION THREAD

**Discussion by dangelsaurus** [weather-gov/api#625] [Q&A]
I noticed that https://api.weather.gov/points/46.5085,-114.7705 shows "**timeZone": "America/Boise"**, however this location is north of the border of the Boise timezone, and should in fact be **America/Los_Angeles**

https://www.openstreetmap.org/relation/6496665#map=7/45.105/-112.808

I was thinking of using the listed timezone to translate forecast to local time, but now I'm curious what this value actually indicates.

**Comment by sullynole** [comment]
That timezone is actually the first timezone found for the zone associated with the point.  In this case the forecast zone is split between Mountain and Pacific and the Mountain zone is returned.  It isn't common for public zones to span across timezoness, but it does happen in a few locations which can be found in the public zone shapefile here: https://www.weather.gov/gis/PublicZones or here https://mapservices.weather.noaa.gov/static/rest/services/nws_reference_maps/nws_reference_map/MapServer
